gooddata-sdk/docs/hugo/docs/content/en/docs/administration/users/get_user.md

```diff
@@ -6,16 +6,25 @@ no_list: true
 superheading: "catalog_user."
 ---
 
-<!-- TODO -->
+
 
 ``get_user(user_id: str)``
 
-Returns *CatalogUser*.
+Gets an individual user using User id.
```

Comment:
Why is there a User and not a user? Below, there are examples e.g. data sources that are all chars lowercase. Could we unify this, please?